ibm_catalog.json

```
@@ -1176,58 +1176,61 @@
     },
     {
       "name": "deploy-arch-ibm-slz-ocp",
-      "label": "Red Hat OpenShift Container Platform on VPC landing zone",
+      "label": "Landing zone for containerized applications with Red Hat OpenShift",
```

You need to drop the "Red Hat" as it exceeds 60 chars
```suggestion
      "label": "Landing zone for containerized applications with OpenShift",
```